diff --git a/dist/_chunks/GoogleMapInput-82adRQ2s.js b/dist/_chunks/GoogleMapInput-82adRQ2s.js
index 0faf31b41aa19c5c7c77bc1d75290b02a0ceff2b..e2b6d8259514f87d7850d9e549638f61fac111d3 100644
--- a/dist/_chunks/GoogleMapInput-82adRQ2s.js
+++ b/dist/_chunks/GoogleMapInput-82adRQ2s.js
@@ -20,7 +20,7 @@ const GoogleMapInput = ({ name, value = {}, onChange, error, description, attrib
   const loadGoogleMapsScript = (key) => new Promise((resolve) => {
     if (window.google && window.google.maps) return resolve();
     const script = document.createElement("script");
-    script.src = `https://maps.googleapis.com/maps/api/js?key=${key}`;
+    script.src = `https://maps.googleapis.com/maps/api/js?key=${key}&libraries=marker`;
     script.async = true;
     script.onload = resolve;
     document.body.appendChild(script);
diff --git a/dist/_chunks/GoogleMapInput-DXoEl3dB.mjs b/dist/_chunks/GoogleMapInput-DXoEl3dB.mjs
index 5f08d39dc9f957093b9d2ef2b21f285807f59373..6927c578bcab6d0e95529877f75497f9365229f8 100644
--- a/dist/_chunks/GoogleMapInput-DXoEl3dB.mjs
+++ b/dist/_chunks/GoogleMapInput-DXoEl3dB.mjs
@@ -18,7 +18,7 @@ const GoogleMapInput = ({ name, value = {}, onChange, error, description, attrib
   const loadGoogleMapsScript = (key) => new Promise((resolve) => {
     if (window.google && window.google.maps) return resolve();
     const script = document.createElement("script");
-    script.src = `https://maps.googleapis.com/maps/api/js?key=${key}`;
+    script.src = `https://maps.googleapis.com/maps/api/js?key=${key}&libraries=marker`;
     script.async = true;
     script.onload = resolve;
     document.body.appendChild(script);
